lib/components/chat_history/twitch/cheer_event.dart

```diff
@@ -5,9 +5,9 @@ import 'package:rtchat/models/messages/twitch/event.dart';
 import 'package:rtchat/models/style.dart';
 
 String getCorrespondingImageUrl(int bits) {
```

Comment:
Can we add some unit tests for this? It also seems simpler to write something like:

```
final key = [10, 100, ...].firstWhere((k) => k < bits, orElse: () => 100000);
return 'https://.../$key/...';
```

Reply:
added unit tests and made it simpler